plugins/action/mocks/action_mock.h

```diff
@@ -0,0 +1,18 @@
+#include <gmock/gmock.h>
+
+// TODO remove this include!
```

Comment:
That's the place were it really feels wrong to include `action.h`, so `action_result.h` would be much nicer.

Reply:
Right, I see.